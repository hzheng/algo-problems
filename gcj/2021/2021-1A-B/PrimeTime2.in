100
5
2 2
3 1
5 2
7 1
11 1
2
2 2
3 1
1
17 2
1
2 7
17
2 9
3 5
7 1
17 1
31 1
239 2
271 1
347 1
349 2
383 1
389 10
401 1
433 3
443 7
449 1
467 2
479 23
17
11 2
13 1
19 2
29 1
31 1
73 1
359 1
379 1
389 6
419 21
421 30
433 7
457 1
461 3
467 1
479 1
491 20
17
2 3
23 1
47 1
197 1
271 1
311 1
317 1
347 1
359 1
397 1
401 10
409 2
433 5
443 37
461 5
487 1
499 1
14
2 5
3 1
251 1
367 1
389 1
401 2
419 1
433 3
439 5
443 37
449 2
461 5
479 1
499 4
17
3 3
5 5
7 1
11 1
19 1
23 1
269 1
349 1
383 43
401 1
409 6
419 6
421 2
431 5
449 18
479 1
491 4
12
2 11
3 1
13 1
19 2
53 1
71 1
293 1
421 2
449 1
487 1
491 61
499 3
16
2 3
37 1
41 1
139 1
151 1
211 1
383 1
433 5
439 2
443 2
461 3
463 6
467 14
479 15
491 11
499 3
63
5 1
7 1
17 1
23 1
29 1
31 1
37 1
41 1
43 1
73 1
83 1
97 1
103 1
109 1
113 1
137 1
139 1
149 1
151 1
163 1
181 1
191 2
197 1
211 1
223 1
227 1
229 1
239 1
251 1
257 1
263 1
271 1
277 1
281 1
311 1
313 1
317 1
331 1
337 1
347 1
349 1
353 1
359 1
367 1
373 1
379 1
383 1
389 1
397 2
409 1
419 1
421 1
431 1
439 1
443 1
449 1
457 1
461 1
463 1
479 2
487 2
491 1
499 1
12
2 3
13 2
31 1
47 1
67 1
293 1
383 1
419 1
461 1
463 1
491 80
499 1
18
2 5
107 1
109 1
163 1
283 1
317 1
359 1
379 17
383 2
401 5
421 4
431 4
433 4
439 2
443 1
463 26
467 11
491 11
17
2 5
5 2
17 2
47 1
109 1
163 1
283 1
347 1
379 2
409 19
419 1
431 1
433 7
443 4
457 5
463 7
487 31
16
2 8
3 1
23 2
29 1
47 1
101 1
331 1
401 16
419 20
421 1
431 4
457 4
463 1
467 4
487 2
499 12
19
2 1
3 9
5 1
137 1
223 1
229 1
263 1
281 12
293 3
347 1
359 1
373 16
383 4
409 3
421 1
433 4
443 3
457 2
499 35
12
2 10
109 1
163 1
263 1
419 1
433 24
439 1
461 1
467 5
479 25
491 4
499 1
5
2 6
5 1
11 1
439 1
467 1
21
2 6
7 7
71 1
97 1
233 1
271 1
277 2
281 1
347 1
353 3
359 1
367 2
389 11
397 7
409 1
433 8
457 1
461 27
487 3
491 9
499 3
54
3 1
7 1
11 1
13 1
17 1
23 1
37 1
43 1
47 1
53 1
59 1
61 1
71 1
73 1
79 1
83 1
89 1
97 1
101 1
103 1
107 1
113 1
127 1
131 1
137 2
139 1
151 2
163 1
167 1
173 1
179 1
191 2
193 1
197 1
199 1
227 1
239 1
251 1
257 1
269 1
271 1
281 2
283 1
293 1
307 2
311 2
313 1
317 1
331 1
349 2
353 2
359 1
367 1
379 1
17
2 5
3 7
5 1
17 2
89 1
101 1
181 1
269 1
307 1
347 4
379 16
397 4
409 1
431 34
433 1
467 16
499 4
10
3 3
113 1
421 2
439 1
457 1
467 31
479 1
487 43
491 3
499 13
13
2 1
3 1
61 1
73 1
359 1
433 3
439 3
443 8
463 1
467 3
487 5
491 3
499 31
18
2 2
43 1
73 1
193 1
227 1
263 2
269 2
293 1
317 1
349 1
367 1
379 3
389 1
401 44
409 1
457 7
461 1
487 27
2
97 1
499 98
5
2 6
3 1
41 1
431 1
487 1
5
2 9
7 1
193 1
359 1
421 1
17
2 1
11 1
41 1
71 1
79 1
307 1
347 1
397 2
421 1
431 8
439 22
443 22
457 1
461 4
467 2
491 1
499 7
6
2 3
37 1
109 1
293 1
457 1
499 63
14
229 1
233 1
337 1
347 1
379 2
401 1
419 4
421 1
461 2
463 1
467 5
479 12
487 55
499 12
1
491 2
12
2 5
3 5
5 1
7 3
239 1
389 3
409 1
433 2
457 29
479 26
491 1
499 20
18
3 1
7 1
41 1
79 1
173 1
311 1
317 1
347 1
389 1
397 3
421 3
443 1
449 1
463 4
467 15
487 17
491 1
499 32
19
2 4
3 5
5 3
7 2
11 7
13 7
17 3
19 1
23 5
29 2
31 5
37 5
41 3
43 4
47 3
53 4
59 5
61 4
67 2
5
2 7
23 1
197 1
257 1
491 1
17
2 3
3 4
5 1
11 1
17 1
19 1
151 1
197 1
227 1
347 1
397 5
421 6
439 18
449 19
457 6
491 9
499 2
15
2 7
3 3
7 7
127 1
373 1
401 3
409 1
419 3
421 10
443 1
449 11
457 6
461 1
479 30
499 11
45
2 2
3 1
5 3
7 3
11 3
13 1
17 2
23 1
29 2
31 1
37 1
41 2
43 2
53 2
59 2
67 1
73 2
79 3
83 1
89 1
97 2
101 2
103 2
107 2
113 1
127 1
149 2
151 2
157 3
163 2
167 3
173 2
179 3
181 1
191 2
193 2
197 3
199 2
229 2
239 2
241 2
251 1
257 2
263 2
269 1
15
2 14
59 1
269 1
331 1
337 2
367 1
373 3
389 8
401 1
433 3
449 3
461 29
463 2
467 1
479 9
11
2 1
3 2
7 2
29 1
67 1
151 1
317 1
439 47
457 1
461 3
479 14
7
2 19
7 1
31 1
197 1
311 1
479 16
491 50
14
2 1
3 1
5 5
23 1
43 2
313 1
383 1
389 8
421 1
449 8
457 25
461 15
467 22
479 2
16
2 1
3 11
5 1
23 1
113 1
137 1
211 1
313 1
431 4
433 1
449 1
461 1
467 35
487 6
491 4
499 2
7
2 1
3 13
5 30
7 2
11 1
19 1
29 49
12
2 2
3 1
7 1
31 1
37 2
193 2
313 1
331 1
389 2
463 11
467 47
487 1
14
2 2
3 2
5 1
23 1
109 1
367 1
439 1
443 1
449 2
457 2
461 2
467 1
479 7
491 76
10
2 11
3 1
47 1
277 1
409 1
431 1
463 12
467 54
491 8
499 7
10
2 2
67 1
163 1
439 23
443 11
449 1
457 47
467 5
479 2
499 7
8
2 5
3 1
29 1
97 1
433 1
461 48
479 23
491 9
12
2 5
5 11
7 4
23 1
199 1
379 1
401 1
449 1
457 1
467 1
479 42
487 17
8
313 1
431 22
433 38
439 16
463 2
487 17
491 2
499 1
6
2 4
7 2
11 4
13 55
17 26
29 9
11
2 1
3 1
5 1
13 2
19 1
113 1
167 1
397 1
463 29
487 15
491 32
15
3 1
11 1
23 1
41 1
167 1
397 5
409 1
431 8
433 1
439 1
443 11
461 1
463 60
487 2
491 2
5
2 15
7 1
31 1
439 1
467 1
18
2 2
7 1
53 1
131 1
271 1
307 1
347 1
379 1
383 13
389 2
397 3
401 1
419 1
443 9
457 1
461 7
479 26
487 16
7
2 5
19 1
89 1
233 1
349 2
491 61
499 1
16
2 14
3 6
41 1
211 2
331 1
359 3
389 8
409 8
419 4
421 3
431 1
439 12
443 14
449 4
461 5
463 12
16
2 5
5 1
13 1
163 1
271 1
277 1
337 4
353 3
359 4
389 1
431 1
461 15
463 2
467 8
479 19
487 21
14
2 13
7 1
17 2
31 1
197 1
281 1
317 1
367 2
379 1
431 1
457 16
479 6
491 1
499 50
18
2 1
3 1
5 1
7 3
11 1
101 1
149 1
283 1
389 1
401 2
419 1
431 1
439 1
461 3
467 34
479 4
487 41
499 2
14
2 3
17 1
229 1
271 1
389 21
397 3
401 7
419 8
439 9
461 1
467 3
479 3
491 8
499 1
5
2 3
3 1
179 1
193 1
499 62
10
2 23
3 3
5 2
7 1
61 1
199 1
311 1
419 1
457 3
499 52
3
2 3
5 2
491 2
11
2 14
3 1
107 1
313 1
317 1
373 1
443 11
457 10
467 26
479 2
487 21
15
3 1
13 6
41 1
83 1
233 1
311 1
347 1
397 1
431 2
443 2
449 11
467 9
479 5
487 54
499 4
31
2 1
3 1
5 1
7 3
11 1
13 2
17 3
19 2
23 2
31 1
37 2
41 2
43 2
47 3
53 3
59 1
61 1
71 1
73 2
79 1
83 1
97 2
101 1
103 3
109 2
113 1
131 1
137 3
139 2
149 2
151 1
1
443 2
14
2 1
5 3
151 1
199 1
223 1
263 1
367 1
379 1
409 2
431 5
443 1
467 9
479 1
487 50
15
2 5
71 1
101 1
127 1
131 1
269 17
283 2
293 9
337 1
367 2
373 1
389 1
401 1
431 36
433 1
95
2 2
3 2
5 2
7 2
11 2
13 1
17 1
19 1
23 1
29 1
31 1
37 1
41 1
43 1
47 1
53 1
59 1
61 1
67 1
71 1
73 1
79 1
83 1
89 1
97 1
101 1
103 1
107 1
109 1
113 1
127 1
131 1
137 1
139 1
149 1
151 1
157 1
163 1
167 1
173 1
179 1
181 1
191 1
193 1
197 1
199 1
211 1
223 1
227 1
229 1
233 1
239 1
241 1
251 1
257 1
263 1
269 1
271 1
277 1
281 1
283 1
293 1
307 1
311 1
313 1
317 1
331 1
337 1
347 1
349 1
353 1
359 1
367 1
373 1
379 1
383 1
389 1
397 1
401 1
409 1
419 1
421 1
431 1
433 1
439 1
443 1
449 1
457 1
461 1
463 1
467 1
479 1
487 1
491 1
499 1
20
3 8
5 3
11 2
19 1
23 1
271 1
307 3
353 1
359 7
367 1
379 1
401 1
419 4
439 2
443 1
457 19
463 2
479 2
491 39
499 1
17
2 1
3 4
11 3
23 1
101 1
109 1
199 1
241 1
251 1
307 2
359 2
379 1
401 1
421 3
463 20
467 41
487 1
7
2 1
7 2
457 33
461 14
463 1
479 3
487 46
10
2 1
5 1
19 1
241 1
449 7
457 18
461 6
463 20
479 3
499 42
16
2 4
5 1
19 1
67 1
191 2
347 1
349 1
383 1
397 1
401 1
443 4
457 27
461 1
467 9
479 4
499 21
16
2 1
3 1
13 5
19 1
233 2
313 1
337 1
347 8
359 2
367 1
373 1
379 1
457 3
467 2
487 66
491 4
15
2 16
59 1
113 1
163 1
193 1
281 1
349 2
397 4
431 1
457 3
461 28
463 18
467 16
479 1
499 6
3
2 4
491 1
499 1
8
2 1
7 1
179 1
379 1
409 1
419 1
431 2
491 7
6
2 10
13 2
137 1
167 1
211 1
293 1
2
443 1
491 1
14
2 4
5 2
131 1
307 1
311 2
359 1
383 1
421 1
457 11
461 21
463 6
467 4
479 3
491 1
10
2 4
3 1
17 1
337 1
401 2
439 1
443 8
461 12
487 2
491 47
10
97 1
107 1
373 1
389 1
409 3
431 1
461 6
463 10
467 56
479 1
15
2 1
3 1
5 6
157 1
263 1
331 1
389 4
409 3
419 1
433 1
439 1
443 22
457 1
463 26
487 30
11
2 3
5 4
7 1
281 1
283 19
337 2
379 25
409 26
421 1
449 14
463 4
4
2 5
3 1
11 1
487 2
19
2 2
3 2
5 1
19 1
89 1
139 1
151 1
347 1
353 5
367 1
389 2
431 2
433 9
443 2
457 1
461 37
467 8
479 16
487 3
8
421 1
431 12
433 50
439 17
467 6
479 1
491 11
499 1
6
2 1
3 1
7 2
113 1
383 1
467 1
11
2 10
3 6
43 1
83 1
379 1
421 1
433 1
439 1
463 4
467 9
499 65
1
73 2
18
2 2
3 2
5 1
29 2
61 1
109 1
151 1
167 3
211 2
419 1
431 1
433 6
443 12
449 1
457 4
461 9
491 17
499 6
19
2 1
7 2
47 1
61 1
83 1
151 1
163 1
331 1
349 1
379 1
389 1
397 1
439 3
449 1
457 1
467 2
479 47
487 3
499 30
21
2 3
3 2
5 2
7 1
37 1
197 2
277 16
283 1
293 1
359 1
367 7
373 6
379 4
389 10
421 8
433 21
457 1
467 4
479 6
487 1
499 1
15
2 1
7 2
13 1
107 1
109 1
283 2
353 1
389 1
409 10
433 2
439 1
467 10
487 11
491 3
499 49
14
2 10
3 3
5 2
7 1
11 1
19 4
199 1
401 1
433 9
449 1
463 6
467 17
487 6
499 32
