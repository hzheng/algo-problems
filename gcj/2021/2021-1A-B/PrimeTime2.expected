Case #1: 25
Case #2: 0
Case #3: 17
Case #4: 8
Case #5: 0
Case #6: 40223
Case #7: 28612
Case #8: 0
Case #9: 35777
Case #10: 33402
Case #11: 29329
Case #12: 0
Case #13: 41912
Case #14: 39131
Case #15: 35534
Case #16: 28482
Case #17: 36031
Case #18: 28667
Case #19: 467
Case #20: 34823
Case #21: 0
Case #22: 34992
Case #23: 0
Case #24: 0
Case #25: 39044
Case #26: 48403
Case #27: 487
Case #28: 0
Case #29: 32021
Case #30: 31937
Case #31: 0
Case #32: 491
Case #33: 38880
Case #34: 38253
Case #35: 2100
Case #36: 491
Case #37: 29747
Case #38: 0
Case #39: 0
Case #40: 27553
Case #41: 0
Case #42: 32768
Case #43: 0
Case #44: 25990
Case #45: 0
Case #46: 29109
Case #47: 45126
Case #48: 39057
Case #49: 43684
Case #50: 37671
Case #51: 0
Case #52: 0
Case #53: 1456
Case #54: 0
Case #55: 0
Case #56: 0
Case #57: 36782
Case #58: 31061
Case #59: 0
Case #60: 36010
Case #61: 37723
Case #62: 42790
Case #63: 0
Case #64: 0
Case #65: 0
Case #66: 0
Case #67: 33491
Case #68: 42681
Case #69: 0
Case #70: 443
Case #71: 33673
Case #72: 0
Case #73: 21437
Case #74: 38095
Case #75: 33027
Case #76: 45374
Case #77: 45790
Case #78: 33155
Case #79: 41838
Case #80: 36838
Case #81: 499
Case #82: 5306
Case #83: 0
Case #84: 0
Case #85: 23050
Case #86: 34374
Case #87: 36181
Case #88: 41291
Case #89: 35000
Case #90: 0
Case #91: 0
Case #92: 0
Case #93: 0
Case #94: 39818
Case #95: 73
Case #96: 27633
Case #97: 44002
Case #98: 0
Case #99: 42401
Case #100: 34650
